OSDOCS-16949#CQA work Stor5 - CSI OpenStack Manila driver (#114605)
openshift/openshift-docs: modules/persistent-storage-csi-manila-overview.adoc

// Module included in the following assemblies:
//
// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
//
:_mod-docs-content-type: CONCEPT

[id="persistent-storage-csi-manila-overview_{context}"]
= Manila CSI Driver Operator Overview

[role="_abstract"]
{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for the OpenStack Manila shared file system service.

> Familiarity with persistent storage] and configuring CSI volumes is recommended when working with CSI Operator and driver. For more information, see _Understanding persistent storage_ and _Configuring CSI volumes_ under _Additional resources_.

Reviewer: ```suggestion
Familiarity with persistent storage] and configuring CSI volumes is recommended when working with CSI Operator and driver. For more information, see "Understanding persistent storage" and "Configuring CSI volumes"  in the "Additional resources" section.
```
please update to current style and accessibility standards

To create CSI-provisioned PVs that mount to Manila storage assets, {product-title} installs the Manila CSI Driver Operator and the Manila CSI driver by default on any OpenStack cluster that has the Manila service enabled.

> * The _Manila CSI Driver Operator_ creates the required storage class that is needed to create persistent volumes claims (PVCs) for all available Manila share types. The Operator is installed in the `openshift-cluster-csi-drivers` namespace.

Reviewer: ```suggestion
* The Manila CSI Driver Operator creates the required storage class that is needed to create persistent volumes claims (PVCs) for all available Manila share types. The Operator is installed in the `openshift-cluster-csi-drivers` namespace.
```

> * The _Manila CSI driver_ enables you to create and mount Manila PVs. The driver is installed in the `openshift-manila-csi-driver` namespace.

Reviewer: ```suggestion
* The Manila CSI driver enables you to create and mount Manila PVs. The driver is installed in the `openshift-manila-csi-driver` namespace.
```
